src/hdateformat.c

```diff
@@ -20,8 +20,8 @@ or connect to: http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
 #include "stpncpy.h"
 #endif
 
-const char* hchar[]={ "׆", "א", "ב", "ג", "ד", "ה", "ו", "ז", "ח", "ט", "י", "כ", "ל", "מ", "נ", "ס", "ע", "פ", "צ", "ק", "ר", "ש", "ת", "״", "׳"};
-const char* hmonth[]={ "אדר א׳", "ניסן", "אייר", "סיון", "תמוז", "אב", "אלול", "תשרי", "חשון", "כסלו", "טבת", "שבט", "אדר", "אדר ב׳"};
+const char* hchar[]={ "׆", "א", "ב", "ג", "ד", "ה", "ו", "ז", "ח", "ט", "י", "כ", "ל", "מ", "נ", "ס", "ע", "פ", "צ", "ק", "ר", "ש", "ת", "\"", "'"};
+const char* hmonth[]={ "אדר א'", "ניסן", "אייר", "סיון", "תמוז", "אב", "אלול", "תשרי", "חשון", "כסלו", "טבת", "שבט", "אדר", "אדר ב'"};
```

Comment:
I find the geresh & gershaim better,

Reply:
> I would prefer not to have arduino specific files in the repo. The library is designed with a 32 bit int (most modern platforms). to adapt it to a 16 bit int would need to change many of the signatures to guarantee correctness, not just the few you changed

so far it has worked splendidly with what i have right now. 
currently arduino doesn't have memory to use the zmanim.c, so i plan to port it to esp32 which i believe is a 32bit library.

Reply:
> I find the geresh & gershaim better,

unfortuently those don't display well, in LCD, the alternative the way i see it is to have a completely different set of strings just for embedded, but then you'd have to maintain 2 sets.